#### RNA-seq data pre-processing
##### RNAseq on CIA paws
CIA paws were sequenced in house on a NextSeq 550 instrument (Illumina) using the NextSeq 500/550 High Output Kit (150 cycles). The cDNA libraries of all samples were first pooled, and then sequenced on 5 runs.

The FASTQ files from each run can be found at: s3://snf-mgln-immunometabolism/CIA_FH2000/RNAseq/illuminaOutput

After each run is completed, the data from 4 lanes was concatenated. 

FASTQ files generated by sequencing mouse paws were pre-processed by RNA-Seq Analysis Snakemake Workflow (RASflow). 

##### RNAseq on human macrophages
FASTQ files generated by sequencing human macrophages were imported into ArrayStudio (Qiagen). Briefly, raw data quality control was performed and then a filtering step was applied to remove reads corresponding to rRNAs as well as reads having low quality score or shorter than 25 nt. Reads were further mapped to the Human genome B38 using Omicsoft sequence aligner (OSA, version 4) and quantified using Ensembl.R96 gene model. Paired reads were counted at the gene level. 

##### RNAseq on human synoviums
For the public RNA-seq dataset on human synovial biopsies (GSE89408), raw counts at the gene level were extracted from HumanDisease_B38_GC33 of DiseaseLand (Qiagen). Briefly, DiseaseLand contains curated publicly available RNA-seq and expression array data, where all samples have been processed through the same pipeline to allow for cross project comparison. For HumanDisease_B38_GC33, alignment of FASTQ files was done using OSA to Human genome B38.

#### DESeq2 and GSEA analysis
##### Figure 3 RNAseq on CIA paws
[Prepare DESeq2 colData](https://github.com/tAndreani/MultiOmics_RA/blob/main/Codes/Analysis_DESeq2_coldataPrep.R)

[Construct a DESeqDataSet](https://github.com/tAndreani/MultiOmics_RA/blob/main/Codes/Analysis_DESeq2_ddsConstruction.R)

[DESeq2 remove batch effect and vst transformation](https://github.com/tAndreani/MultiOmics_RA/blob/main/Codes/Analysis_DESeq2_rm_Batch_vst.R)

[DESeq2 statistical test](https://github.com/tAndreani/MultiOmics_RA/blob/main/Codes/Analysis_DESeq2_LRT.R)

[Annotation of ENSEMBL gene IDs](https://github.com/tAndreani/MultiOmics_RA/blob/main/Codes/Analysis_geneAnnotation.R)

[GSEA](https://github.com/tAndreani/MultiOmics_RA/blob/main/Codes/Analysis_GSEA_GO.R)

##### Figure 6 RNAseq on human macrophages
[Construct a DESeqDataSet](https://github.com/tAndreani/MultiOmics_RA/blob/main/Codes/Normoxia_Analysis_DESeq2_ddsConstruction.R)

[DESeq2 vst transformation](https://github.com/tAndreani/MultiOmics_RA/blob/main/Codes/Normoxia_PCA_DESeq2_Normalization.R)

[DESeq2 statistical test](https://github.com/tAndreani/MultiOmics_RA/blob/main/Codes/Normoxia_Analysis_DESeq2_Wald_Test.R)

[Annotation of ENSEMBL gene IDs](https://github.com/tAndreani/MultiOmics_RA/blob/main/Codes/Analysis_geneAnnotation_hs.R)

[GSEA](https://github.com/tAndreani/MultiOmics_RA/blob/main/Codes/Analysis_GSEA_GO_BP_hs.R)

##### Figure 7 RNAseq on human synoviums
[Construct a DESeqDataSet](https://github.com/tAndreani/MultiOmics_RA/blob/main/Codes/Analysis_GSE89408_DESeq2_ddsConstruction.R)

[DESeq2 vst transformation](https://github.com/tAndreani/MultiOmics_RA/blob/main/Codes/PCA_GSE89408_DESeq2_Normalization.R)

[DESeq2 statistical test](https://github.com/tAndreani/MultiOmics_RA/blob/main/Codes/Analysis_GSE89408_DESeq2_Wald_Test.R)

[Annotation of ENSEMBL gene IDs](https://github.com/tAndreani/MultiOmics_RA/blob/main/Codes/Analysis_GSE89408_geneAnnotation.R)

[GSEA](https://github.com/tAndreani/MultiOmics_RA/blob/main/Codes/Analysis_GSE89408_GSEA_GO_BP_hs.R)
